selfdrive/ui/ui.cc

```diff
@@ -118,7 +118,12 @@ static void update_model(UIState *s, const cereal::ModelDataV2::Reader &model) {
 }
 
 static void update_sockets(UIState *s) {
-  s->sm->update(0);
+  // ensures UI stays responsive when modelV2 is not alive
+  int timeout = s->sm->alive("modelV2") ? 1000 / UI_FREQ : 0;
```

Comment:
UI was much more jittery when scrolling in any of the views when just setting to 50 when model was dead, even with an instant singleshot timer